src/main/java/com/esri/samples/scene/sync_map_and_scene_viewpoints/README.md

```diff
@@ -0,0 +1,40 @@
+<h1>Sync map and scene viewpoints</h1>
+
+<p>Synchronize the view points between a <code>MapView</code> and a <code>SceneView</code>.</p>
+
+<p>The two <code>GeoViews</code> share a common <code>ViewPoint</code>. When navigating in one view, the other view is immediately updated to display the same <code>ViewPoint</code>, resulting in a continuously updated and synchronized view.</p>
+
+<p><img src="SyncMapandSceneViewpoints.png" /></p>
```

Comment:
For the image, It would be good to zoom out on the scene view so it shows the Earth's curvature. Then you don't need the labels.